libfwupd/fwupd.map

```diff
@@ -447,6 +447,15 @@ LIBFWUPD_1.4.1 {
   local: *;
 } LIBFWUPD_1.4.0;
 
+LIBFWUPD_1.4.5 {
```

Comment:
1.4.5?

Reply:
I wondered if you would notice that; my intention for this would be to backport the new API into 1_4_X so that we could include it in Fedora 32 etc. I think 1.5.0 is some way off being "ready" as I think the HSI spec kinda determines when we need to branch again.

Reply:
Hmm, new API in the stable branch..  I guess it's not a _super_ risky API.

>I think 1.5.0 is some way off being "ready" as I think the HSI spec kinda determines when we need to branch again.

I'm of the perspective we should probably release 1.5.0 soon.  Lots of changes that should get flushed out and HSI will stay _pre-release_ in 1.5.0.  When we're ready to have it live, drop the need for `--force`

Reply:
I'd like to get the ARM and AMD opinions before we commit to API.

Reply:
Oh you mean because of library implications of potentially breaking API.   Is there really that much that can change from another set of vendors coming on board?  I'm curious what sort of things you envision could have to happen.

Reply:
> Oh you mean because of library implications of potentially breaking API

Right.

>  Is there really that much that can change from another set of vendors coming on board

I'm not really sure. I wouldn't be at all surprised if there were some kind of random requirement on displaying the results for instance.